types/culori/index.d.ts

```diff
@@ -0,0 +1,282 @@
+// Type definitions for culori 2.0
+// Project: https://github.com/Evercoder/culori
+// Definitions by: Bijela Gora <https://github.com/bijela-gora>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="./fn/index.d.ts" />
```

Comment:
so this line is here just to make sure `culori/fn` symbols are exported?
If so, you can consider removing this one, adding 'OTHER_FILES.TXT' file and referencing `fn/index.d.ts` in it:
https://github.com/DefinitelyTyped/DefinitelyTyped#other_filestxt

Reply:
I just removed `/// <reference path="./fn/index.d.ts" />` line.